.eslintrc.yml

```diff
@@ -0,0 +1,32 @@
+env:
+  commonjs: true
+  node: true
+extends: 'eslint:recommended'
+rules:
+  indent:
+    - error
+    - tab
```

Comment:
Does it mean we need to change all the code-base from spaces to tabs? Is there a compelling reason to do so?

Reply:
Hm, that's a mistake I did during the setup. I was pretty sure I selected spaces 😅 

I made sure the rules were the same as the ones configured for JSHint.